src/test/java/uk/gov/hmcts/cmc/claimstore/models/sampledata/SampleTheirDetails.java

```diff
@@ -16,6 +16,7 @@
 
     private String name = "John Smith";
     private Address address = SampleAddress.validDefaults();
+    private Address serviceAddress = SampleAddress.validDefaults();
```

Comment:
Does this need to default to something?
Its an optional field so I would have thought it would be null by default